src/controllers/contractNames/contractNames.test.ts

```diff
@@ -73,30 +73,38 @@ describe('Contract Names', () => {
     // request a non contract address that will fail
     contractNamesController.getName(randomAddress, 8453n)
 
-    // The default time for debounce is 50ms
-    jest.advanceTimersByTime(50)
+    // The default time for debounce is 100ms
+    jest.advanceTimersByTime(100)
```

Comment:
If that's a constant from the logic, take it from there instead of hardcoding it?